plugin/teksi_wastewater/metadata.txt

```diff
@@ -0,0 +1,24 @@
+
+[general]
+name=TEKSI Wastewater
+description=Plugin to manage wastewater networks (formerly: QGEP) # // skip-keyword-check
+qgisMinimumVersion=3.2
+category=Vector
+version=dev
+
+changelog=See repository for full changelog
+tags=teksi, gep, abwasser, wastewater, qgep, assainissement # // skip-keyword-check
+author=TEKSI
+email=info@teksi.ch
+about=The TEKSI wastewater plugin adds functionality to manage wastewater networks in QGIS. It was formerly known as QGEP. # // skip-keyword-check
+homepage=https://www.teksi.ch/modules/abwasser-gep/
+tracker=https://github.com/teksi/teksi_wastewater/issues
+repository=https://github.com/teksi/teksi_wastewater
+icon=icons/teksi-abwasser-logo.svg
+
+experimental=False
+deprecated=False
+
+# DISABLED: under QGIS 3.22, this throws `KeyError: 'use_stable_version' (qgsplugindependenciesdialog.py", line 63)`
+# unfortunately this doesn't support >= yet...
```

Comment:
is this fixed in 3.28? we could raise the min version accordingly.

Reply:
in fact is it no more needed to have the model baker plugin installed as we can use now the model baker library